OpenEdXMobile/res/layout/fragment_discussion_responses_or_comments.xml

```diff
@@ -20,12 +18,8 @@
             android:layout_weight="1" />
 
         <include layout="@layout/create_new_item_layout" />
-
     </LinearLayout>
-
-
```

Comment:
Only 1 newline should've been removed.

Pro-tip: Always auto-format a class/layout file whenever you make some changes, that way you don't have to care about things like these.